docs/readmes/resources/pcap.md

```diff
@@ -0,0 +1,39 @@
+---
+id: pcap
+title: PCAP Collection
+hide_title: true
+---
+
+*Last Updated: 4/20/2021*
+
+# PCAP collection
+
+This is a collection of `.pcap` files that can be open using [Wireshark](https://www.wireshark.org/). Some of those PCAP collect data
```

Comment:
`Some of those PCAP collect data` seems like an incomplete sentence :D

Reply:
good catch! it was hidden on my IDE far far in the right